### PR TITLE
fix: unresponsive add page

### LINKED ISSUE #4506: Unresponsive Add Page for the File, Image and Link
**The page for adding File, Image, and Link is unresponsive for mobile devices**

**To Reproduce**
This can be seen in changing the device width to 375px or similar.

**Expected behavior**
The Page should be responsive for all device widths.

**Screenshots**

**_Before_**

Screenshot 2023-03-10 at 3 56 53 PM

**_After_**
Screenshot 2023-03-12 at 2 54 55 PM

**Software:**

- OS: any
- Browser: any

### PR DESCRIPTION
**Pull request by sudhanshu1309**
The tabs  for the add page was unresponsive on mobile devices. Fixed this by changing flex-wrap property
closes #4506

### CODE REVIEW

File: theme/themes/pastanaga/extras/sidebar.less
Comment (stevepiercy) [inline theme/themes/pastanaga/extras/sidebar.less:495]:
```suggestion
}

```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
GitHub likes a final line break as an EOF indicator, else it shows a red circle with a slash through it.

Also delete extra space.

File: news/4506.bugfix
Comment (stevepiercy) [inline news/4506.bugfix:1]: I don't know why this was missed.

```suggestion
The tabs for the add page was unresponsive on mobile devices. Fixed this by changing flex-wrap property. @sudhanshu1309
```
```suggestion
The tabs  for the add page was unresponsive on mobile devices. Fixed this by changing flex-wrap property. @sudhanshu1309
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
This needs review by a core Volto developer before merging.

**Review by sneridagh — APPROVED** [review]
I guess this is good to go.

**Pull request merged**
